onnxruntime/core/providers/cpu/quantization/quantize_linear.cc

```diff
@@ -599,6 +599,20 @@ Status DequantizeLinear<T>::Compute(OpKernelContext* ctx) const {
           .TypeConstraint("T2", DataTypeImpl::GetTensorType<T>()),          \
       QuantizeLinear<T>);
 
+#define REGISTER_QUANTIZELINEAR_VERSIONED_POST_22(T)                        \
+  ONNX_CPU_OPERATOR_VERSIONED_TYPED_KERNEL(                                 \
+      QuantizeLinear,                                                       \
+      23,                                                                   \
+      23,                                                                   \
```

Comment:
We do not want macros contain fixed versions.
The latest kernel should have open ended version